tests-and-examples/tests/cases/auth_context.rs

```diff
@@ -23,19 +24,22 @@ impl Greeter for GreeterService {
         sink: UnarySink<HelloReply>,
     ) {
         let auth_context = ctx.auth_context();
-        self.tx
-            .send((
-                "AuthContextPresent".to_string(),
-                (if auth_context.is_some() { "Y" } else { "N" }).to_string(),
-            ))
-            .unwrap();
+
+        if auth_context.is_some() {
+            self.tx.send(Ok(None)).unwrap();
```

Comment:
This is duplicated with L42.

Reply:
This line is to ensure auth_context.is_some() 
FYI https://github.com/tikv/grpc-rs/blob/master/tests-and-examples/tests/cases/auth_context.rs#L29

Reply:
L42 has proven it's some already.

Reply:
Fine i'll fix